src/getopt.h

```diff
@@ -0,0 +1,20 @@
+#ifndef GETOPT_H
+#define GETOPT_H 1
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+extern int opterr;
+extern int optind;
+extern int optopt;
+extern int optreset;
+extern char *optarg;
```

Comment:
opterr, optind, optopt, optreset, optarg is global variables under Linux by including `unistd.h`. If src/webpng.c or any other source file want to access them, there should be a header file define them as global vars.